devenv/dev-dashboards/panel-bargauge/panel_tests_bar_gauge.json

```diff
@@ -143,8 +142,7 @@
             "steps": [
               {
                 "color": "blue",
-                "index": 0,
-                "value": null
```

Comment:
We needed to run `scripts/stripnulls.sh` (uugggghhhhhhhhh how is this still a thing)